src/components/Tabs.vue

```diff
@@ -3,41 +3,43 @@
 </template>
 
 <script>
+import i18n from '@/i18n';
+
 export default {
   props: ['pool'],
   computed: {
     items() {
       const items = [
         {
-          name: 'Balances',
+          name: i18n.tc('balances'),
```

Comment:
You could do it like this:
```suggestion
          name: this.$t('balances'),
```
This is more consistent with the way you do it on the view template (`$t('key')`) it also avoid importing `@/i18n`

Reply:
Thanks! I struggled with that - finally figured it out, but didn't think to try this.$t